.circleci/config.yml

```diff
@@ -268,6 +271,123 @@ jobs:
+          command: openssl aes-256-cbc -md sha256 -d -in .circleci/.firebase.secrets.json.enc -out .circleci/.firebase.secrets.json -k "${FIREBASE_SECRETS_ENCRYPTION_KEY}"
+      # - android/firebase-test:
+      #     key-file: .circleci/.firebase.secrets.json
+      #     type: instrumentation
+      #     apk-path: WordPress/build/outputs/apk/vanilla/debug/org.wordpress.android-vanilla-debug.apk
+      #     test-apk-path: WordPress/build/outputs/apk/androidTest/vanilla/debug/org.wordpress.android-vanilla-debug-androidTest.apk
+      #     test-targets: package org.wordpress.android.ui.screenshots
+      #     devices: |
+      #       model=blueline,version=28,orientation=portrait
+      #       model=gts3lltevzw,version=28,orientation=default
+      #     # [Trial Limitation] Free plan I use to experiment with Firebase Test Lab has a quota of 10 tests/day
+      #     # locales: ar de_DE en_US es_ES fr_FR iw_IL in it_IT ja_JP ko_KR nl_NL pl_PL pt_BR ru_RU sr sv_SE th tr_TR vi zh_CN zh_TW
+      #     locales: en_US in
+      #     project: wpandroid-74b6c
+      #     timeout: 30m
+      #     num-flaky-test-attempts: 2
+      #     results-history-name: CircleCI WordPress Screenshots
+      # - android/save-gradle-cache
+
+      # BEGIN DEBUG STEP (While firebase-test is commented out)
+      - run:
+          name: Fake FTL Run
+          command: |
+            gcloud auth activate-service-account --key-file ".circleci/.firebase.secrets.json"
+            echo "INFO: Raw results root path is: [gs://test-lab-d1urt75q3y6wu-ha5s1mih364hi/2020-08-26_08:51:14.975190_umlG/]" >log.txt
+      # END DEBUG STEP
+
+      - run:
+          name: Download Raw Screenshots from Google Storage
+          command: |
+            TEST_BUCKET=$(cat log.txt | grep -o "gs://test\-lab\-.*/" | head -1)
+            bundle exec fastlane download_raw_screenshots bucket:"$TEST_BUCKET" phone:blueline-28 tenInch:gts3lltevzw-28
+      - persist_to_workspace:
+          root: .
+          paths:
+            - Gemfile
+            - Gemfile.lock
+            - fastlane/
+  promo-screenshots:
+    parameters:
+      post-to-slack:
+        description: Post to Slack when screenshots job has finished. SLACK_WEBHOOK ENV variable must be set.
+        type: boolean
+        default: true
+    macos:
+      xcode: 11.6.0
+    working_directory: /tmp/workspace
+    steps:
+      - attach_workspace:
+          at: .
+      - restore_cache:
+          # ImageMagick is used by the 'rmagick' gem, so cache invalidation of ImageMagick should depend on version of rmagick from Gemfile.lock
+          keys:
+            - &homebrew-cache-key v2-brew-imagemagick-{{ checksum "Gemfile.lock" }}
+      - run:
+          name: Brew Install ImageMagick
+          command: |
+            brew install pkg-config imagemagick
+            brew cleanup
+      - save_cache:
+          paths:
+            - ~/Library/Caches/Homebrew
+          key: *homebrew-cache-key
```

Comment:
tbh, I'm not that sure that this saves us that much time – the only thing we can cache with Homebrew is the downloaded bottles, but we still need to re-install the bottles on each run (as it would be a bad idea to cache the whole content of `/usr/local` where the formulae are installed by brew) – so I'm wondering if it's worth keeping?